api.js

```diff
@@ -119,7 +126,7 @@ function HttpError(url, opts, response) {
     this.status = response.status;
     this.opts = opts;
     this.response = response;
-};
+}
```

Comment:
Are these changes from some additional lintin tool you're using, or from the npm run-script style tool?  Just want to be consistent and not keep reformatting back-and-forth.  If there's a linting tool you like we should add it as a script that everyone can run.

Happy to do more linting so long as we can ignore certain things via a  config file - this javascript runs in a weird environment.